### PR TITLE
Add info about creation of rt tokens

### PR DESCRIPTION
**Pull request by johannaengland**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,6 +25,8 @@ to support V2, while RT 5.0.0 and later has the support included.
     }
     ```
 
+    A token for RT can be created under "Logged in as ..." -> "Settings" -> "Auth tokens".
```
Comment (hmpf) [inline README.md]: This doesn't seem to be turned on by default in an RT installation so we'll have to mention a bit more: That if this is missing, the functionally is not available in that installation of RT, maybe with a link to how to turn it on.

Also, I'm assuming the username/password is what you get when not using SSO? If there is a way to make a new user just for API access we should probably point towards how as well.

**Review by hmpf — CHANGES_REQUESTED** [review]
Mention that the Token-functionality might not be available.

File: README.md (hunk 1 above)
Comment (johannaengland) [reply README.md]: Also here I should add that in case of creating a new user to make sure it has access to the relevant queue.

Comment (hmpf) [reply README.md:28]: Oh yeah.

Comment (hmpf) [reply README.md]: Btw, turns out that the user-based api-token can be turned on per user, not just per site. No way for a regular user to see the difference.

**Review by hmpf — APPROVED** [review]

**Pull request merged**